index.html

```diff
@@ -14,7 +14,46 @@
   <header class="header-container">
     <a class="mylogo" href="#">My Logo</a>
     <nav class="menu">
-      <img src="image/menu.png" alt="Menu Icon">
+      <img src="icons/menu.png" alt="Menu Icon">
     </nav>
   </header>
-</body>
\ No newline at end of file
+  <main class="home">
+    <section class="bio">
+      <h1 class="title">I’m Jawad
+        Glad to see you!</h1>
+      <p class="text">
+        I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+        experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="social">
+        <h3 class="connect">LET'S CONNECT</h3>
+        <ul>
+          <li><a href="https://twitter.com/Nazari_Jawad58">
+              <img src="icons/twitter.svg" alt="twitter">
+            </a>
+          </li>
+          <li><a href="https://www.linkedin.com/in/jawad-nazari/">
+              <img src="icons/Linkedin icon.svg" alt="Linkedin">
+            </a>
+          </li>
+          <li><a href="#">
+              <img src="icons/M.png" alt="M">
+            </a>
+          </li>
+          <li><a href="https://github.com/Jawad-Nazari">
+              <img src="icons/github.svg" alt="github">
+            </a>
+          </li>
+          <li><a href="#">
+              <img src="icons/angellist.svg" alt="angellist">
+            </a>
+          </li>
+        </ul>
```

Comment:
- 💃 💃 You did it! 💯 Everything seems alright now except that the background image is not showing.
- [ ] Kindly make the background image show correctly to match your first look or the figma design as contained [here](https://github.com/Jawad-Nazari/Portfolio-Mobile-1/pull/1/files#r1112271801).

See  this 👇 
![image](https://user-images.githubusercontent.com/99715640/220332495-d5987d22-8709-431b-bb5b-bb6decfc2919.png)